lib/fast_jsonapi/object_serializer.rb

```diff
@@ -245,18 +245,20 @@ def create_relationship(base_key, relationship_type, options, block)
         polymorphic = fetch_polymorphic_option(options)
 
         Relationship.new(
+          owner: self,
           key: options[:key] || run_key_transform(base_key),
           name: name,
           id_method_name: compute_id_method_name(
             options[:id_method_name],
             "#{base_serialization_key}#{id_postfix}".to_sym,
             polymorphic,
+            options[:serializer],
             block
           ),
-          record_type: options[:record_type] || run_key_transform(base_key_sym),
+          record_type: options[:record_type],
```

Comment:
Yes, I created #34 to discuss deprecating `record_type`.  For now, I wanted to keep this PR's impact to a minimum.  We can't remove it without deprecating it because it will be a breaking change.  For now, if `record_type` is specified, it should be used, period.  That is how it worked before, so that is how it should remain until we remove the option all together.

Reply:
Alrighty, fair, let's do it separately.